### DISCUSSION TITLE
dolphinscheduler3.1.7点击项目管理后一直转圈，不能加载出项目

### DISCUSSION THREAD

**Discussion by 966699** [apache/dolphinscheduler#16896] [Q&A]
dolphinscheduler3.1.7点击项目管理后一直转圈，不能加载出项目，前面几天偶尔能加载出来，最近两三天大多就不能加载了。
此项目已经运行了一年多，采用集群模式，1master，3worker，问题是最近一周才出现的，以前都正常。项目加载不了但是项目里面任务的调度正常运行。请各位帮忙分析一下原因？清理过浏览器缓存，更换过浏览器，重启过应用和数据库。
![image](https://github.com/user-attachments/assets/cb33b052-d86e-450c-9d66-d9c3f20050fc)
master报错日志
[master-error.txt](https://github.com/user-attachments/files/18119353/master-error.txt)

**Comment by Axiangg** [comment]
creating index can solve  it:
`CREATE INDEX t_ds_process_instance_process_definition_code_idx ON public.t_ds_process_instance (process_definition_code,state,process_definition_version);`
